examples/spin-timer/app-example/src/lib.rs

```diff
@@ -1,9 +1,7 @@
 wit_bindgen::generate!({
     world: "spin-timer",
     path: "..",
-    exports: {
-        world: MySpinTimer
-    }
+    generate_all,
 });
 
```

Comment:
This file is missing `export!(MySpinTimer)`

Reply:
So it does!